# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/streams/readable-byte-streams/patched-global.any.js.ini
# Commit: 7d2880a833f4
# Full Hash: 7d2880a833f4422b1efa336353e3843801ea3f70
# Author: Alexandra Borovova <aborovova@mozilla.com>
# Date: 2024-12-11 10:02:50
# Description:
#   Bug 1934354 - Update test expectations for streams/readable-byte-streams/patched-global.any.js. r=aryx
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231687
# ==============================================================================

diff -r 76f964932238 -r 7d2880a833f4 testing/web-platform/meta/streams/readable-byte-streams/patched-global.any.js.ini
--- a/testing/web-platform/meta/streams/readable-byte-streams/patched-global.any.js.ini	Tue Dec 10 16:13:58 2024 +0000
+++ b/testing/web-platform/meta/streams/readable-byte-streams/patched-global.any.js.ini	Tue Dec 10 16:26:12 2024 +0000
@@ -18,7 +18,7 @@
 
 [patched-global.any.serviceworker.html]
   expected:
-    if not debug: OK
+    if not debug: [OK, ERROR, CRASH]
     [ERROR, CRASH]
   [Patched then() sees byobRequest after filling all pending pull-into descriptors]
     expected: FAIL
@@ -46,10 +46,4 @@
   expected: [ERROR, CRASH]
 
 [patched-global.https.any.shadowrealm-in-serviceworker.html]
-  expected:
-    if (os == "linux") and debug and fission: [TIMEOUT, ERROR]
-    if (os == "linux") and debug and not fission: [TIMEOUT, ERROR]
-    if (os == "android") and sessionHistoryInParent and debug: [ERROR, TIMEOUT]
-    if (os == "android") and sessionHistoryInParent and not debug: [ERROR, TIMEOUT]
-    if (os == "android") and not sessionHistoryInParent and debug: [ERROR, TIMEOUT]
-    ERROR
+  expected: [ERROR, TIMEOUT]
